src/GraphQL/Types/Schema.cs

```diff
@@ -208,6 +223,18 @@ public IGraphType FindType(string name)
             return _lookup?.Value[name];
         }
 
+        public IFieldType GetIntrospectionField<T>()
```

Comment:
Maybe switch expression?

Reply:
Unfortunately impossible due to `typeof(T)` is not a constant

Reply:
However, we could exchange this method for three separate fields, as was done for #1564 which was pulled in earlier today.